examples/long_running/README.md

```diff
@@ -1,14 +1,14 @@
-# Example model specifications
+# Example model specification
 
-## Choosing an example
+## Example Stats
```

Comment:
It does not make much sense to have a table with one entry. I would keep the old structure, but move it up by one level.

Further helicopter-level notes:
- harmonise the way how the models are run (probably keep a `simulate_model.py` (renamed `run_model.py`) in each of them?
- Rename `long_running` to something more descriptive. At least I'd expect `mahler_yum_2024` to run even longer?
- Can we access `examples` when running the tests? In that case, we should move the test models over here.